osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-monitoring.xsd

```diff
@@ -540,4 +540,103 @@
     </xsd:sequence>
   </xsd:complexType>
 
+  <!-- ActualPowerQuality -->
+
+  <xsd:element name="ActualPowerQualityRequest">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="tns:ActualPowerQualityRequestData">
+          <xsd:sequence>
+            <xsd:element name="DeviceIdentification" type="common:DeviceIdentification"/>
+          </xsd:sequence>
+        </xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:complexType name="ActualPowerQualityRequestData">
+    <xsd:complexContent>
+      <xsd:extension base="common:Query">
+        <xsd:sequence>
+          <xsd:element name="ProfileType" type="xsd:string"/>
```

Comment:
If the values are intentionally kept open, this could be OK.
I have a feeling though, that it might be a bit more client-friendly to restrict this to only the values "PUBLIC" or "PRIVATE".

Reply:
Already used Strings in earlier implementation for ProfileType. Would have large impact on existing code